Project2015To2017.Migrate2017.Library/Project2015To2017.Migrate2017.Library.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Revert indentation here and in all non-feature changed places, please.

Reply:
It's my ctrl+k+d habit, difficult to kick. It would be nice for a future PR if we can run dotnet format automatically across the solution and then it won't happen again. I'll try and back these ones out

Reply:
Following my formatting check-in, there is just the one formatting change to the top of one of the project files. For a future PR we should find a way to standardise csproj format too as some of them have a mix of tabs and spaces at the moment

Reply:
I would expect the .editorconfig to pick that up that's not just for cs files